static/app/views/dashboards/widgets/timeSeriesWidget/useReleaseStats.tsx

```diff
@@ -8,11 +9,28 @@ interface ReleaseMetaBasic {
   version: string;
 }
 
+interface UseReleaseStatsParams {
+  datetime: Parameters<typeof normalizeDateTimeParams>[0];
+  environments: readonly string[];
+  projects: readonly number[];
+
+  /**
+   * Max number of pages to fetch. Default is 10 pages, which should be
+   * sufficient to fetch "all" releases.
+   */
+  maxPages?: number;
```

Comment:
Also decided to add this so we can control the # of pages to fetch and to limit the damage if something were to go wrong or if an org has more releases than we expect. For reference, Sentry fetches ~3 pages for a 90d window.